generator/Generator/ModelGenerator.php

```diff
@@ -137,6 +137,10 @@ public function generate(): void
                     );
 
                     [$newNamespaceDirectory, $className] = $this->getNamespaceFromPath($path, $httpMethod);
+                    if (empty($className) === true) {
```

Comment:
note: extra precaution in case endpoints might have no operationId or summary, like old/not used `/compute` script endpoints.